src/components/Tasks.tsx

```diff
@@ -1,18 +1,180 @@
-import React from 'react';
+import React, { useEffect, useState, useRef } from 'react';
+import deleteImage from '../assets/img/delete.png';
```

Comment:
prepend unused variables with an underscore
```suggestion
import _deleteImage from '../assets/img/delete.png';
```